contribs/dvrp/src/main/java/org/matsim/contrib/dvrp/path/VrpPaths.java

```diff
@@ -98,18 +123,25 @@ public static VrpPathWithTravelData createPath(Link fromLink, Link toLink, doubl
 
 		// there is no extra time spent on queuing at the end of the last link
 		links[count + 1] = toLink;
-		linkTT = getLastLinkTT(toLink, currentTime);// as long as we cannot divert from the last link this is okay
```

Comment:
@michalmac Before getLastLinkTT (which only uses freeflow travel time) was used here, now I use the time coming from the TravelTime object

Reply:
But I left the getLastLinkTT function there, because otherwise I'd need to change a lot of other code. Basically, I would need to pass down the TravelTime object to the code that creates travel time graphs which involves changing many different classes. Not sure if it is worth it at this point, but we could also do that

Reply:
That's the point, if I want to do exactly the same thing, I need to pass `TravelTime` to getLastLinkTT. But this means I need to pass down `TravelTime` down to all the places where getLastLinkTT is used. This requires passing on `TravelTime` through the constructors of quite some classes, so I have the feeling that some resdesign of some classes that create the travel time graphs etc. may be cleaner (and some of the uses of getLastLinkTT were not completely clear to me, I think). So that's why I would deprecate getLastLinkTT for now and look into this in a later step. What do you say?

Reply:
Deprecating is also fine with me. This can be addressed anytime later.